index.js

```diff
@@ -11,36 +11,17 @@ import generatePdfForSubscriber from './utils/pdfgeneration';
 import EmailGenerators from './utils/emailCopy';
 import sendEmail from './utils/email';
 import { getEnphaseDataForMonth } from './utils/enphase';
-import { getSolarProjectById, updateSolarProject } from './airtable/request';
+import {
+  getSolarProjectById,
+  updateSolarProject,
+  getOwnersByEmail
+} from './airtable/request';
 import Constants from './Constants';
 
-const AppConfigurations = [
-  'ENPHASE_KEY',
-  'REACT_APP_AIRTABLE_API_KEY',
-  'UTILITY_API_KEY',
-  'MAIL_SERVER_EMAIL',
-  'MAIL_SERVER_PASS',
-  'ADMIN_EMAIL',
-  'AIRTABLE_BASE_ID',
-  'AIRTABLE_ENDPOINT_URL',
-  'PRODUCTION_WEB_URL',
-  'SERVER_URL',
-  'SENDER_NAME',
-  'ACCEPT_HIGHCHARTS_LICENSE'
-];
-
-// Fails explicitly if the environment is improperly configured
-AppConfigurations.forEach(param => {
-  if (!process.env[param]) {
-    throw new Error(
-      `Required configuration variable ${param} is ${process.env[param]}. Do you have a .env file and is it setup correctly?`
-    );
-  }
-});
-
```

Comment:
is there a reason you're opposed to this? I personally think it's a good idea for the app to fail explicitly, so we know from the get-go why  it hasn't been properly configured, as opposed to it throwing up silent errors later on

Reply:
the app DOES fail explicitly. The package I used to load environment variables `dotenv-safe` makes sure that all the variables that are in .env.example are loaded. Otherwise the app won’t start.

Reply:
not opposed at all, we just already have it :)